docs/demo/demo.md

```diff
@@ -122,7 +120,6 @@ To see a visualization of the relationships between tables and views,
 expand the following, which shows table names in lowercase and view
 names in uppercase:
 
-<details><summary>E-R diagram of tables and views</summary>
```

Comment:
so this text is lost?

Reply:
I don't know how to support this line in the new format, but I don't think it's very important